core/src/main/scala/org/apache/spark/deploy/Client.scala

```diff
@@ -221,7 +221,9 @@ object Client {
     val conf = new SparkConf()
     val driverArgs = new ClientArguments(args)
 
-    conf.set("spark.rpc.askTimeout", "10")
+    if (!conf.contains("spark.rpc.askTimeout")) {
```

Comment:
also we should probably add this to `config/package.scala` so we don't duplicate it